br/pkg/metautil/metafile.go

```diff
@@ -146,6 +146,12 @@ func (tbl *Table) NoChecksum() bool {
 	return tbl.Crc64Xor == 0 && tbl.TotalKvs == 0 && tbl.TotalBytes == 0
 }
 
+// FullName generates the full table name like schema.table.
+func (tbl *Table) FullName() string {
+	// we cannot use `utils.EncloseDBAndTable(tbl.DB.Name.L, tbl.Info.Name.L)` because of cyclic requirement...
+	return fmt.Sprintf("`%s`.`%s`", tbl.DB.Name, tbl.Info.Name)
+}
```

Comment:
since this won't escape the `` ` `` i think it's better in stream.go:694 use explicitly ``Sprintf("... `%s`.`%s`", table.DB.Name, table.Info.Name)``